app/v2/models/responses/hateoas/HateoasDepartureIdsResponse.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.responses.hateoas
+
+import play.api.libs.json._
+import v2.models.DepartureId
+
+object HateoasDepartureIdsResponse extends HateoasResponse {
+
+  def apply(departureIds: Seq[DepartureId]): JsObject =
+    Json.obj(
+      "_links" -> Json.obj(
+        "self" -> Json.obj("href" -> "/customs/transits/movements/departures")
+      ),
+      "ids" -> departureIds.map(
+        id => Json.obj("href" -> departureUri(id))
```

Comment:
We should match what we're doing with message IDs here, so:

```
"departures": [
   {
      "id": "/customs/transits/movements/departures/<id>"
   }
]
```

not what you have here.

Reply:
Agree. Will change.